data/SageKyle.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ominyi Paul, Saje",
+  "displayStatsPublic": true,
```

Comment:
We recently have removed the public stats feature. So `displayStatsPublic` property is no longer needed. 
Please remove this from your json. 

```suggestion
```